sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -210,12 +232,16 @@ export class EventProcessor {
   private _partitionManager: PartitionManager;
 
   /**
-   * @param consumerGroupName The consumer group name used in this event processor to consumer events.
-   * @param eventHubAsyncClient The Event Hub client.
-   * @param partitionProcessorFactory The factory to create new partition processor(s).
-   * @param initialEventPosition Initial event position to start consuming events.
-   * @param partitionManager The partition manager.
-   * @param eventHubName The Event Hub name.
+   * @param consumerGroupName The name of the consumer group from which you want to process events
+   * @param eventHubAsyncClient An instance of `EventHubClient` that was created for the Event Hub instance.
```

Comment:
I know you didn't make the original param, but I think `eventHubAsyncClient` should be `eventHubClient`.